Containerfile.static

```diff
@@ -0,0 +1,12 @@
+FROM debian:testing
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y \
+    make \
+    golang \
+    git-core \
+    libglib2.0-dev
+
+ENV GOPATH=/
+WORKDIR /src/github.com/containers/libpod
```

Comment:
Did you intend to `ADD . /src/github.com/containers/libpod` and then `RUN podman-remote-static-local`?

Reply:
No we volume mont that in and execute a podman or docker command to build the static binary.